hackrf-portapack/hackrf_ui.c

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2019-2022 Great Scott Gadgets <info@greatscottgadgets.com>
```

Comment:
Shouldn't you add your own note here ?

Reply:
nah, it's just some tiny changes in the copied file.